charts/tuf/values.yaml

```diff
@@ -35,7 +35,7 @@ secrets:
   tsa:
     create: false
     name: tsa-cert-chain
-    key: cert-chain
+    key: key
```

Comment:
Would it be possible to update the tsa chart to read cert-chain rather than key? That would be more accurate.

Reply:
yep, addressed in https://github.com/sigstore/helm-charts/pull/712/files. i'll close this pr.